src/main/java/com/dnd/namuiwiki/domain/wiki/WikiService.java

```diff
@@ -0,0 +1,52 @@
+package com.dnd.namuiwiki.domain.wiki;
+
+import com.dnd.namuiwiki.domain.jwt.JwtService;
+import com.dnd.namuiwiki.domain.question.QuestionRepository;
+import com.dnd.namuiwiki.domain.survey.SurveyRepository;
+import com.dnd.namuiwiki.domain.user.entity.User;
+import com.dnd.namuiwiki.domain.wiki.dto.GetWikisResponse;
+import com.dnd.namuiwiki.domain.wiki.dto.WikiDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class WikiService {
+    private final JwtService jwtService;
+    private final QuestionRepository questionRepository;
+    private final SurveyRepository surveyRepository;
+
+    public GetWikisResponse getWikis(String accessToken) {
+        Long namuiWikiQuestionCount = questionRepository.countByWikiType(WikiType.NAMUI);
+        Long romanceWikiQuestionCount = questionRepository.countByWikiType(WikiType.RAMANCE);
+
+        WikiDto namuiWiki = WikiDto.builder()
+                .wikiType(WikiType.NAMUI)
+                .name(WikiType.NAMUI.getTitle())
+                .description(WikiType.NAMUI.getDescription())
+                .questionCount(namuiWikiQuestionCount)
+                .build();
+        WikiDto romanceWiki = WikiDto.builder()
+                .wikiType(WikiType.RAMANCE)
+                .name(WikiType.RAMANCE.getTitle())
+                .description(WikiType.RAMANCE.getDescription())
+                .questionCount(romanceWikiQuestionCount)
+                .build();
+
+        List<WikiDto> wikiList = new ArrayList<>();
```

Comment:
```suggestion
        List<WikiDto> wikiList = Arrays.stream(WikiType.values()).map(wikiType -> {
            Long wikiQuestionCount = questionRepository.countByWikiType(wikiType);
            return WikiDto.builder()
                    .wikiType(wikiType)
                    .name(wikiType.getTitle())
                    .description(wikiType.getDescription())
                    .questionCount(wikiQuestionCount)
                    .build();
        }).collect(Collectors.toList());
```

`values()` 사용하도록 바꾸면 어떤가요?
아래 accessToken도 map 함수 안에 넣어도 될 것 같기도 한데, 외부 변수를 사용하는게 조금 걸리긴 하네요.
accessToken 있을 때와 없을 때 나눠서 아예 함수로 분리해도 좋을 것 같아요!